inspirehep/modules/authors/rest/publications.py

```diff
@@ -96,14 +97,17 @@ def serialize(self, pid, record, links_factory=None):
             try:
                 publication['journal'] = {}
                 publication['journal']['title'] = result_source.get(
-                    'publication_info', [])[0]['journal_title']
+                    'publication_info', []
```

Comment:
For example:

```python
something(
    'string, with commas, and', something, 'that', (' makes, (it' + really, really), hard, ')to read',
) 
# vs
something(
    'string, with commas, and',
    something,
    'that',
    (' makes, (it' + really, really),
    hard,
    ')to read',
)
```

On the first you have completely no idea how many params are there even with syntax highlight.